test/unit/range/iterator_test_template.hpp

```diff
@@ -45,8 +45,10 @@ struct iterator_fixture : public ::testing::Test
 
 // Helper concept to check whether the test fixture has a member function expect_eq.
 template <typename T>
-SEQAN3_CONCEPT HasExpectEqualMemberFunction = requires(T a) {
```

Comment:
But we use this in a lot of places don't we?

Does this change maybe fix this:

```suggestion
SEQAN3_CONCEPT HasExpectEqualMemberFunction = requires(T & a) {
```
I think I saw this in other places, to explitily state that this is `a` lvalue